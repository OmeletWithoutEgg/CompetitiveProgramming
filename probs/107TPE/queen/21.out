209301.49 26717527.32
