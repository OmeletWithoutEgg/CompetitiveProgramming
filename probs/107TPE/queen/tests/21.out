-427305.84 -35754092.33
